app/views/handlers/list_add_question.py

```diff
@@ -21,8 +21,9 @@ def handle_post(self):
     def _resolve_custom_page_title_vars(self) -> MutableMapping:
         # For list add blocks, no list item id is yet available. Instead, we resolve
         # `list_item_position` to the position in the list it would be if added.
-        list_length = len(
-            self._questionnaire_store.list_store[self._current_location.list_name]
-        )
+        if self._current_location.list_name:
+            list_length = len(
+                self._questionnaire_store.list_store[self._current_location.list_name]
+            )
 
-        return {"list_item_position": list_length + 1}
+            return {"list_item_position": list_length + 1}
```

Comment:
Same here, list name should exist, if it didn't, currently runner would error. So we either need to handle that, or ignore type hinting and accept the risk that we should have configured it right.

Reply:
Added ignore to type hinting here, I hope you agree?